roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -97,12 +97,39 @@ def get_var(self, *keys, **kwargs):
         OpenShiftCheckException when a key is not found or returning a default value
         provided as a keyword argument.
         """
+        if len(keys) == 1:
+            keys = keys[0].split(".")
+
         try:
             value = reduce(operator.getitem, keys, self.task_vars)
         except (KeyError, TypeError):
             if "default" in kwargs:
                 return kwargs["default"]
             raise OpenShiftCheckException("'{}' is undefined".format(".".join(map(str, keys))))
+
+        convert = kwargs.get("type")
```

Comment:
> note the logic under bool...

Hmm, why would we add that YAML-like conversion here?

For `bool`, any non-empty string converts to `True`.

Now this gets me thinking that "covert" is normally from A to B, can we say that A is always of type `str`?